sdk/storage/Azure.Storage.Blobs/src/BlobExtensions.cs

```diff
@@ -1321,5 +1321,18 @@ internal static BlobContainerProperties ToBlobContainerProperties(this ResponseW
             return metadata;
         }
         #endregion
+
+        #region CreateConditionsWithEtag
+        internal static BlobRequestConditions CreateConditionsWithEtag(this BlobRequestConditions conditions, ETag etag) =>
```

Comment:
Why not put this on the `BlobRequestConditions` class?  Then it's closer to any new members that get added so it's more likely to be updated.  And why not call it `WithIfMatch` to match the new wither pattern in C# 9?

Reply:
will do.